tests/generate_packageTest.py

```diff
@@ -90,7 +90,7 @@ class MainTest(unittest.TestCase):
     NO_BEAR_PATH = os.path.join('bears', 'BadBear', 'InvalidBear.py')
 
     def setUp(self):
-        self.argv = ['generate_package.py']
+        self.argv = ['generate_package.py -b CSSLintBear']
```

Comment:
This should probably be changed to `['generate_package.py', '-b',  'CSSLintBear']`